### PR TITLE
Home,End,PageUp,PageDown keys functionality

### PR DESCRIPTION
**Pull request by elig0n**

### CODE REVIEW

File: Switcheroo/MainWindow.xaml [hunk 1]
```diff
@@ -65,7 +81,7 @@
                     </TextBlock>
                 </StackPanel>
             </DockPanel>
-            <Grid DockPanel.Dock="Top">
+            <Grid Name="grd0" DockPanel.Dock="Top">
```
Comment (kvakulo) [inline Switcheroo/MainWindow.xaml]: Could we come up with a better name for the grid? 😄

File: Switcheroo/MainWindow.xaml.cs [hunk 2]
```diff
@@ -683,6 +689,49 @@ private void NextItem()
             }
         }
 
+        private void ScrollListPageUp(object sender, ExecutedRoutedEventArgs e)
+        {
+            var n = Math.Round(lb.ActualHeight / grd0.ActualHeight);
```
Comment (kvakulo) [inline Switcheroo/MainWindow.xaml.cs]: I would like if this was extracted into a method with an explanatory name

**Review by kvakulo — CHANGES_REQUESTED** [review]
Thanks for this PR as well! 💯 

I'm ready to merge this into master if you could adjust the code a little bit 😸

File: Switcheroo/MainWindow.xaml (hunk 1 above)
Comment (elig0n) [reply Switcheroo/MainWindow.xaml]: Sure. What do you have in mind ?

Comment (kvakulo) [reply Switcheroo/MainWindow.xaml]: Maybe "SearchGrid"?

**Comment by kvakulo** [comment]
I'd like to accept this, but it's a bit mixed up with the other PR's e.g. #87, so I can't really accept just the one 😅 

For new PR's please fork the latest master of Switcheroo, and make each change on a separate branch. Then it's a lot easier for me to see which changes you have made, and I can accept each one separately. I'll try to get this one merged into Switcheroo when I've looked a bit closer, and selected the relevant parts.

**Pull request closed without merging**